public/v2/apps/phpldapadmin.json

```diff
@@ -0,0 +1,72 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/osixia/docker-openldap",
+    "description": "OpenLDAP with administration interface",
+    "displayName": "OpenLDAP + phpLDAPadmin",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "notExposeAsWebApp": "true",
+                "environment": {
+                    "LDAP_ORGANISATION": "$$cap_openldap_ldap-organisation",
+                    "LDAP_DOMAIN": "$$cap_root_domain",
+                    "LDAP_ADMIN_PASSWORD": "$$cap_openldap_ldap-admin-password"
+                }
+            },
```

Comment:
One last question - doesn't openldap require any persistent volume? With current implementation it'll lose all its data upon restart.

Reply:
Done :smile: